src/Templates/v3/Projects/WixProject/Product.wxs

```diff
@@ -13,7 +13,7 @@
 
 	<Fragment>
 		<Directory Id="TARGETDIR" Name="SourceDir">
-			<Directory Id="ProgramFilesFolder">
+			<Directory Id="$(var.PlatformProgramFilesFolder)">
```

Comment:
I don't think we should do this. You shouldn't build the project as x86 and as x64 and then distribute them both. You would need to vary other things as well, such as the UpgradeCode.